### PR TITLE
boards: arm: arduino_nano_33_ble: Add i2c sensors to DTS

### PR DESCRIPTION
**Pull request by benjaminbjornsson**
This commit adds the hts221, lps22hb and apds9960 sensors to
the DTS since these sensors are present on the Sense version
of the board.

These changes were verified using the sample of each sensor.

Signed-off-by: Benjamin Björnsson <benjamin.bjornsson@gmail.com>

### CODE REVIEW

**Review by MaureenHelm — DISMISSED** [review]

File: boards/arm/arduino_nano_33_ble/arduino_nano_33_ble.dts [hunk 1]
```diff
@@ -95,6 +95,28 @@
...
+	lps22hb: lps22hb-press@5c {
+		compatible = "st,lps22hb-press";
+		status = "disabled";
+		reg = <0x5c>;
+		label = "LPS22HB";
+	};
+
+	apds9960: apds9960@39 {
+		compatible = "avago,apds9960";
+		status = "disabled";
```
Comment (gmarull) [inline boards/arm/arduino_nano_33_ble/arduino_nano_33_ble.dts:115]: should these be `status = "okay"`? They won't get enabled unless `CONFIG_SENSOR=y` and `CONFIG_XXX=y` (XXX=sensor driver)

Comment (benjaminbjornsson) [reply boards/arm/arduino_nano_33_ble/arduino_nano_33_ble.dts]: Sounds resonable, thank you 👍

**Review by gmarull — DISMISSED** [review]

**Comment by MaureenHelm** [comment]
Please rebase

**Comment by benjaminbjornsson** [comment]
Should I remove the regulator-fixed-sync compatible since it expects no delay? I had to add the startup delay to the vdd_env node to make it work. Haven't been able to verify it myself but this post suggests that the reason for the delay is to make some time for the vdd_env pin to go high: https://devzone.nordicsemi.com/f/nordic-q-a/77845/problem-initializing-i2c-on-arduino-nano-33-ble.

File: boards/arm/arduino_nano_33_ble/arduino_nano_33_ble.dts [hunk 2]
```diff
@@ -49,6 +49,7 @@
 		regulator-name = "vdd_env";
 		enable-gpios = <&gpio0 22 GPIO_ACTIVE_HIGH>;
 		regulator-boot-on;
+		startup-delay-us = <5000>;
```
Comment (gmarull) [inline boards/arm/arduino_nano_33_ble/arduino_nano_33_ble.dts:52]: why this change? it's not described in the commit message.

**Review by gmarull — CHANGES_REQUESTED** [review]
you mention that added sensors are only available on the sense version of the board. where is this board variant defined? Please check other boards that do have variants, e.g. longan_nano.

**Review by gmarull — APPROVED** [review]

**Review by MaureenHelm — APPROVED** [review]
